modules/ossm-kiali-service-mesh.adoc

```diff
@@ -14,4 +13,4 @@ Installing Kiali  via the Service Mesh on {product-title} differs from community
 * Ingress has been enabled by default.
 * Updates have been made to the Kiali ConfigMap.
 * Updates have been made to the ClusterRole settings for Kiali.
-* Do not edit the ConfigMap or the Kiali custom resource files as those changes might be overwritten by the {ProductShortName} or Kiali Operators. All configuration for Kiali running on {ProductName} is done in the `ServiceMeshControlPlane` custom resource file and there are limited configuration options. Updating the Operator files should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the operator files should be restricted to those users with `dedicated-admin` privileges.
+* Do not edit the ConfigMap, as your changes might be overwritten by the {ProductShortName} or Kiali Operators. Files that are managed by the Kiali Operator have a `kiali.io/` label or annotation. Updating the Operator files should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the Operator files should be restricted to those users with `dedicated-admin` privileges.
```

Comment:
1. I don't believe "files" is the correct term to use. Since we are talking about resources that the operator creates (e.g. Deployments, Roles, Services, etc) replace "files" with "resources".
2. When the operator creates resources, it labels them with a label named `app.kubernetes.io/name` whose value is `kiali` (the value is actually the value of the Kiali CR's `spec.deployment.instance_name` but most people just keep it the default).

```suggestion
* Do not edit the ConfigMap, as your changes might be overwritten by the {ProductShortName} or Kiali Operator. Resources that are managed by the Kiali Operator have a `app.kubernetes.io/name` label with value `kiali` (or the value of the Kiali CR's `spec.deployment.instance_name`). Updating the Operator resources should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the Operator resources should be restricted to those users with `dedicated-admin` privileges.
```